pyfixest/estimation/fepois_.py

```diff
@@ -113,6 +115,32 @@ def __init__(
         self.deviance = None
         self._Xbeta = np.array([])
 
+    def solve_ols(self, tZX, tZY, solver):
```

Comment:
It is sufficient to define `solve_ols` as a method for `Feols` - as the other classes (`Feiv`, `Fepois`) are derived from `Feols`, the other classes will inherit all of `Feols` methods.

Reply:
Done with [945f611](https://github.com/py-econometrics/pyfixest/pull/513/commits/945f611a151866f5ee3c187093e41cfb4a385f1f).